### PR TITLE
Local storage

### PR DESCRIPTION
**Pull request by Alexr16**
* Data is saved to the local storage.
* Pre-filled data when the user loads the page.
* JavaScript object used for save form data in local storage.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -54,6 +54,36 @@ form.addEventListener('submit', (event) => {
...
+    'full-name': fullNameForm,
+    email: emailForm,
+    message: messageForm,
+  };
+  localStorage.setItem('user', JSON.stringify(user));
+}
+
+form.addEventListener('change', () => {
+  saveData(userData);
+});
```
Comment (medaminedev66) [inline script.js:83]: - [x] Following the project requirement,  all entered values (Name, Email, Message) should be saved in the local storage after filling the form, **and not submitting it**. If the user reloads the page or closes the browser and opens it again, the form inputs should be auto-populated with the values entered in the previous visit to the page.
- *Please notice that your code doesn't save all the data in the localStorage. Kindly work on that 😃.*

**Review by medaminedev66 — CHANGES_REQUESTED** [review]
Hi @Alexr16 and @Dangelo-JAN, :wave:

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

_See the inline comments below for more required changes_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Alexr16** [comment]
*Added action, save Values (Name, Email, Message) in the local storage after filling the form and not submitting it. 
* Added action, If the user reloads the page or closes the browser and opens it again, the form inputs are auto-populated with the values entered in the previous visit to the page.

File: script.js (hunk 1 above)
Comment (mateo951) [reply script.js:83]: Addressed 🔥

**Review by mateo951 — APPROVED** [review]
### Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 😎 
### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**